pkg/types/defaults/validation/featuregates.go

```diff
@@ -35,32 +35,5 @@ func GatedFeatures(c *types.InstallConfig) []featuregates.GatedInstallConfigFeat
 			}(),
 			Field: field.NewPath("compute", "diskSetup"),
 		},
-		{
-			FeatureGateName: features.FeatureGateNodeSwap,
```

Comment:
We need to change this to be features.FeatureGateMultiDiskSetup.  We chose the incorrect FG when we added this and it was removed without us knowning.  Sorry for this confusion.